mappings/net/minecraft/util/math/floatprovider/ClampedNormalFloatProvider.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_5861 net/minecraft/util/math/floatprovider/ClampedNormalFloatProvider
```

Comment:
`floatprovider` seems awfully verbose as a package name (and also as a class name to a small extent, although I'm mostly okay with it), and judging by the content of these classes I'd prefer something along the lines of `distribution` for these

Reply:
I named the package this way since it's similar tree decorators, foliage placers, and trunk placers. `distribution` may work but it's similar size to `floatprovider` and since it's only float providers, we might as well be more specific

Reply:
I see your point.
I'd also like to bring up that "float provider" is a bit of a general term, and that "distribution" (which may not being the right term) seems a bit more specific and more accurately describes what these classes are used for.
However since it seems like these "float providers" can provide a float by any logic (with a Random as input) I get your reasons for naming it this way. My only issue is how people will interpret the name of this class. I think i'll need another opinion on this.

Reply:
I named it "float provider" as that's what the registry string calls them, to avoid confusion. They could definitely have better names but I feel it's out of our hands :P